src/Val.h

```diff
@@ -376,23 +272,10 @@ class Val : public Obj {
 	static ValPtr MakeInt(bro_int_t i);
 	static ValPtr MakeCount(bro_uint_t u);
 
-	template<typename V>
-	Val(V&& v, TypeTag t) noexcept
```

Comment:
These two constructors need to be marked as deprecated and remain until 4.1 at least, assuming they can be implemented with the new code in mind.

Reply:
If I'm understanding the templating correctly, then I don't see how to do this, as it appears to rely on `Val`'s providing a way to store an underlying value, which they no longer have.  However, maybe I'm just confused - can you show me an example of how these used to be used?

Reply:
Given that we're removing a bunch of other constructors as well, I'm ok with leaving these two out.